go/runtime/host/logger.go

```diff
@@ -22,7 +23,9 @@ const maxLogBufferSize = 10_000
 // It hardcodes some assumptions about the format of the runtime logs.
 type RuntimeLogWrapper struct {
 	// Logger for wrapper-internal info/errors.
-	logger *logging.Logger
+	internalLogger *logging.Logger
+	// Runtime-specific logger that should be used as an additional log sink.
+	runtimeLogger log.Logger
```

Comment:
Feels a bit weird mixing two different loggers (go-kit and the oasis-core package), but I guess you used it due to the simpler interface?

Reply:
Yeah I agree. I'll check if this could be refactored so that we would not need the external interface.

Reply:
Changed, please take another look.

Reply:
Thanks, looks good to me now 👍